libs/ngx-mime/src/lib/viewer/viewer.component.scss

```diff
@@ -69,11 +69,17 @@ mat-drawer {
 }
 
 ::ng-deep .viewer-container .hit {
-  fill: rgba(255, 255, 0, 0.6);
+  fill: #ffff00;
+  stroke: #ff8900;
+  stroke-width: 4px;
+  mix-blend-mode: multiply;
 }
 
 ::ng-deep .viewer-container .selected {
-  fill: rgba(255, 225, 0, 0.6);
+  fill: #ff8900;
+  stroke: rgb(97, 52, 0);
+  stroke-width: 4px;
+  mix-blend-mode: multiply;
```

Comment:
If I'm not mistaken the `selected` class is always set on an element that has class `hit` set? In so case, you don't have to override `stroke-width` and `mix-blend-mode` as they have same values.